app/templates/moveItemProperties.html

```diff
@@ -0,0 +1,78 @@
+<div>
+  <div class="item-header" ng-class="vm.classes">
+
+    <div>
+      <span ng-if="asideHasLockOrElement(vm.item)" class="item-header__aside">
```

Comment:
Using function calls in Angular bound expressions is perf poison! Please avoid this in favor of using simple properties precomputed on the view model or the item.